rest/blip_api_test.go

```diff
@@ -3793,10 +3795,11 @@ func TestMultipleOutstandingChangesSubscriptions(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "", errorCode, "resp: %s", respBody)
 
-	base.WaitForStat(pullStats.NumPullReplTotalOneShot.Value, 1)
-	value, _ := base.WaitForStat(pullStats.NumPullReplActiveOneShot.Value, 0)
-	require.EqualValues(t, 0, value)
-	require.EqualValues(t, 1, pullStats.NumPullReplSinceZero.Value())
+	base.RequireWaitForStat(t, pullStats.NumPullReplActiveOneShot.Value, 0)
+	base.RequireWaitForStat(t, pullStats.NumPullReplTotalOneShot.Value, 1)
```

Comment:
This could cause a race condition if the one shot replication has not had a chance to start and therefore bypassing this check instantly, meaning an active replication could be running when `NumPullReplTotalOneShot` is checked. If this race condition happens, then the one shot replication could still be running when the next continuous sub changes feed is created. 
```suggestion
	base.RequireWaitForStat(t, pullStats.NumPullReplTotalOneShot.Value, 1)
	base.RequireWaitForStat(t, pullStats.NumPullReplActiveOneShot.Value, 0)
```

Reply:
That makes sense, thanks... I'll reverse all of the Total/Active stats in the test.